instances/instance_stop_by_name.go

```diff
@@ -13,15 +13,11 @@ import (
 	"fmt"
 	"net/http"
 
-	"sdk.kraft.cloud/client"
+	kcclient "sdk.kraft.cloud/client"
 )
 
-// StopByName the specified instance based on its name, but does not destroy it. All volatile state
-// (e.g., RAM contents) is lost. Does nothing for an instance that is already
-// stopped. The instance can be started again with the start endpoint.
-//
-// See: https://docs.kraft.cloud/api/v1/instances/#stop
-func (c *instancesClient) StopByName(ctx context.Context, name string, drainTimeoutMS int64) (*Instance, error) {
```

Comment:
Also a comment for everywhere: I liked the comments for each function, even if they were maybe a bit too verbose. We can live without it also I guess.

Reply:
Ah, they are now part of the interface, ok.

Reply:
Right, I realized that comments could very easily get out of sync, especially if they are duplicated in multiple places.
Since users interact with client _interfaces_ and not the concrete `client` types, docstrings are on the interface methods to ease their maintenance.